FluentTc/Locators/BuildParameterCheckboxTypeBuilder.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Text;
+
+namespace FluentTc.Locators
+{
+    public interface IBuildParameterCheckboxTypeBuilder
+    {
+        IBuildParameterCheckboxTypeBuilder WithCheckedValue(string checkedValue);
+        IBuildParameterCheckboxTypeBuilder WithUncheckedValue(string uncheckedValue);
+    }
+
+    internal class BuildParameterCheckboxTypeBuilder : IBuildParameterCheckboxTypeBuilder
+    {
+        private string m_checkedValue;
+        private string m_uncheckedValue;
+
+        public IBuildParameterCheckboxTypeBuilder WithCheckedValue(string checkedValue)
+        {
+            m_checkedValue = checkedValue;
+            return this;
+        }
+
+        public IBuildParameterCheckboxTypeBuilder WithUncheckedValue(string uncheckedValue)
+        {
+            m_uncheckedValue = uncheckedValue;
+            return this;
+        }
+
+        public string Build()
+        {
+            var builder = new StringBuilder();
+            builder.Append("checkbox");
+            if (!string.IsNullOrEmpty(m_checkedValue))
+                builder.Append(" checkedValue='").Append(m_checkedValue).Append("'");
```

Comment:
1. It would be clearer with a single call to AppendFormat can be used instead of several calls to Append. 
2. The format can be a const, like you did in other cases